test/uk/gov/hmrc/tai/service/JrsServiceSpec.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.service
+
+import uk.gov.hmrc.tai.config.ApplicationConfig
+import uk.gov.hmrc.tai.connectors.JrsConnector
+import org.mockito.Mockito.when
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import uk.gov.hmrc.tai.model.{Employers, JrsClaims, YearAndMonth}
+import utils.BaseSpec
+
+import scala.concurrent.Future
+
+class JrsServiceSpec extends BaseSpec with ScalaFutures with IntegrationPatience {
+
+  val jrsConnector = mock[JrsConnector]
+  val mockAppConfig = mock[ApplicationConfig]
+
+  val jrsService = new JrsService(jrsConnector, mockAppConfig)
+
+  when(mockAppConfig.jrsClaimsEnabled).thenReturn(true)
+
+  val jrsClaimsAPIResponse = JrsClaims(
+    List(
+      Employers("TESCO", "ABC-DEFGHIJ", List(YearAndMonth("2020-12"), YearAndMonth("2020-11"))),
+      Employers("ASDA", "ABC-DEFGHIJ", List(YearAndMonth("2021-02"), YearAndMonth("2021-01")))
+    ))
+
+  val jrsClaimsServiceResponse = JrsClaims(
+    List(
+      Employers("ASDA", "ABC-DEFGHIJ", List(YearAndMonth("January 2021"), YearAndMonth("February 2021"))),
+      Employers("TESCO", "ABC-DEFGHIJ", List(YearAndMonth("December 2020")))
+    ))
+
+  "getJrsClaims" should {
+
+    "sort the employers alphabetically and yearAndMonth in ascending order" when {
```

Comment:
More complex sorting tests should live in JrsClaimSpec, which is currently untested

Reply:
Done